cognite/client/data_classes/datapoints_subscriptions.py

```diff
@@ -361,6 +361,10 @@ class DatapointSubscriptionList(CogniteResourceList[DatapointSubscription]):
     _RESOURCE = DatapointSubscription
 
 
+class DatapointSubscriptionMemberList(CogniteResourceList[TimeSeriesID]):
```

Comment:
Lets inherit from the mixing (in the `_base.py`) in enables you to do `.as_external_ids()` and `.as_ids()`
```suggestion
class DatapointSubscriptionMemberList(CogniteResourceList[TimeSeriesID], IdTransformerMixin):
```